optimism/deploy/003_delegated_manager_system.ts

```diff
@@ -0,0 +1,193 @@
+import "module-alias/register";
+
+import { HardhatRuntimeEnvironment as HRE } from "hardhat/types";
+import { DeployFunction } from "hardhat-deploy/types";
+
+import {
+  prepareDeployment,
+  findDependency,
+  getContractAddress,
+  getCurrentStage,
+  saveContractDeployment,
+  stageAlreadyFinished,
+  trackFinishedStage,
+  saveDeferredTransactionData,
+  writeTransactionToOutputs,
+  getAccounts,
+} from "@utils/index";
+
+import { Account } from "@utils/types";
+import { InstanceGetter } from "@utils/instanceGetter";
+
+import { DEPENDENCY } from "../deployments/utils/dependencies";
+import { CONTRACT_NAMES } from "../deployments/constants/003_delegated_manager_system";
+
+const {
+  CONTROLLER,
+  SET_TOKEN_CREATOR,
+  ISSUANCE_MODULE,
+  STREAMING_FEE_MODULE,
+  TRADE_MODULE,
+} = DEPENDENCY;
+
+let owner: Account;
+let instanceGetter: InstanceGetter;
+
+const CURRENT_STAGE = getCurrentStage(__filename);
+
+const func: DeployFunction = trackFinishedStage(CURRENT_STAGE, async function (bre: HRE) {
+  const {
+    deploy,
+    deployer,
+    rawTx,
+    networkConstant
+  } = await prepareDeployment(bre);
+
+  [owner] = await getAccounts();
+  instanceGetter = new InstanceGetter(owner.wallet);
+
+  await deployManagerCore();
+  const managerCoreAddress = await getContractAddress(CONTRACT_NAMES.MANAGER_CORE);
+
+  const controllerAddress = await findDependency(CONTROLLER);
+  const setTokenCreatorAddress = await findDependency(SET_TOKEN_CREATOR);
+  await deployDelegatedManagerFactory();
+  const delegatedManagerFactoryAddress = await getContractAddress(CONTRACT_NAMES.DELEGATED_MANAGER_FACTORY);
+
+  const issuanceModuleAddress = await findDependency(ISSUANCE_MODULE);
+  await deployIssuanceExtension();
+  const issuanceExtensionAddress = await getContractAddress(CONTRACT_NAMES.ISSUANCE_EXTENSION);
+
+  const streamingFeeModuleAddress = await findDependency(STREAMING_FEE_MODULE);
+  await deployStreamingFeeSplitExtension();
+  const streamingFeeSplitExtensionAddress = await getContractAddress(CONTRACT_NAMES.STREAMING_FEE_SPLIT_EXTENSION);
+
+  const tradeModuleAddress = await findDependency(TRADE_MODULE);
+  await deployTradeExtension();
+  const tradeExtensionAddress = await getContractAddress(CONTRACT_NAMES.TRADE_EXTENSION);
+
+  await initializeManagerCore();
```

Comment:
Same open question here about transferring ownership of ManagerCore to the governance multisig for production...I guess Polygon script will need the same update if this is necessary.

Reply:
Added `ManagerCore` ownership transfer to multisig. Also removed unnecessary deferred transaction in `ManagerCore.initialize`.